### ISSUE TITLE
ImportError: cannot import name 'get_document_model' from 'wagtail.documents.models'

### ISSUE THREAD

**Issue by shanewilson** [GrappleGQL/wagtail-grapple#93]
related to #46 

wagtail==2.10
wagtail-grapple==0.5.5

```
 File "/usr/local/lib/python3.8/site-packages/grapple/types/search.py", line 3, in <module>
api_1  |     from wagtail.documents.models import get_document_model
api_1  | ImportError: cannot import name 'get_document_model' from 'wagtail.documents.models' (/usr/local/lib/python3.8/site-packages/wagtail/documents/models.py)
...
 ImportError: Could not import 'grapple.schema.schema' for Graphene setting 'SCHEMA'. ImportError: cannot import name 'get_document_model' from 'wagtail.documents.models'
```

It looks like only one of the two `get_document_model` imports was updated:
https://github.com/GrappleGQL/wagtail-grapple/search?q=get_document_model&unscoped_q=get_document_model

**Comment by zerolab** [comment]
Thank you for this @shanewilson 
Fixed in 569b05d

**Comment by zerolab** [comment]
I've pushed v 0.5.6 to pypi
